README.md

```diff
@@ -155,6 +156,7 @@ The following table shows the supported combinations:
 
 | LiT               | LND          | Loop         | Faraday       | Pool         | Taproot Assets |
 |-------------------|--------------|--------------|---------------|--------------|----------------|
+| **v0.13.0-alpha** | v0.18.0-beta | v0.28.3-beta | v0.2.13-alpha | v0.6.5-beta  | v0.3.3-alpha   |
```

Comment:
Not sure exactly what to set `pool` & `tapd` to here, as those won't be official releases, but rather branches litd will depend on.

Reply:
The latest tag versions are okay IMO. So this looks good.